website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,224 @@
+
+```diff
+//…
+  “devDependencies”: {
+   // …
++  “@react-native-community/cli”: “15.0.0”,
++  "@react-native-community/cli-platform-android": “15.0.0”,
++  "@react-native-community/cli-platform-ios": “15.0.0”,
+  },
+```
+
+### [Android] React Native is distributed as a single library
+
+React Native 0.76 will ship with a reduced number of native dynamic libraries, as we merged a lot of our native code into a single library called `libreactnative.so`.
+This change comes with reduction in app size, and improvement in app startup performance on Android.
+
+On the other hand, this is a breaking change for both app and library developers.
+
+App developers will have to update their Application’s `onCreate` as follows:
+
+```diff
++import com.facebook.react.soloader.OpenSourceMergedSoMapping
+import com.facebook.soloader.SoLoader
+
+class MainApplication : Application(), ReactApplication {
+
+  override fun onCreate() {
+    super.onCreate()
++   SoLoader.init(this, OpenSourceMergedSoMapping)
+-   SoLoader.init(this, false)
+  }
+}
+```
+
+This change is necessary in order to properly load `libreactnative.so` and is included in the upgrade-helper.
+
+Library authors won’t be affected by this change, unless you have custom C++ code.
+For a technical deep dive into this change, and suggestions for library authors, you can read more about it [in this dedicated post](https://github.com/react-native-community/discussions-and-proposals/discussions/816).
+
+### Updates to Minimum iOS and Android SDK requirements
+
+We have updated our minimum platform and SDK versions:
+
+- iOS: [15.1](https://support.apple.com/en-gb/108051#151)
+- Android: [SDK 24](https://developer.android.com/tools/releases/platforms#7.0) (Android 7)
+
+This change was announced earlier in the year when 0.75 [was released](/blog/2024/08/12/release-0.75#last-version-supporting-minsdk-23-and-miniosversion-134). For more background, please see the dedicated posts for [Android](https://github.com/react-native-community/discussions-and-proposals/discussions/802) and [iOS](https://github.com/react-native-community/discussions-and-proposals/discussions/812).
+
+### Other Breaking Changes
+
+- **Animation**
+  - Stop sending state updates to React in looping animation ([commit](https://github.com/facebook/react-native/commit/8af5e893c89b24dca972afe36ef442fd509d5804)). This was causing unnecessary re-rendering with looping animations.
+- **devtools:**
+  - Remove Inspector Panel perf + network tabs under New Arch ([commit](https://github.com/facebook/react-native/commit/f220bde4d7d556b3be83cccff0eec5da22921aba)). ([RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0777-remove-legacy-element-inspector-features.md))
+- **text engine**
+  - Always use AttributedStringBox instead of AttributedString in TextLayoutManager ([commit](https://github.com/facebook/react-native/commit/ee597bfe2bf7f11ac60dab87f47dbcb6e840749c))
```

Comment:
nits:

- Formatting consistency. If there's any chance of adding a second sentence, prefer periods on every line.
- Also, looks like we changed the format of this section in the 0.75 post — are we including "(commit)" any more? 

```suggestion
  - Remove Inspector Panel perf + network tabs under New Arch ([commit](https://github.com/facebook/react-native/commit/f220bde4d7d556b3be83cccff0eec5da22921aba)) ([RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0777-remove-legacy-element-inspector-features.md)).
- **Text engine**
  - Always use AttributedStringBox instead of AttributedString in TextLayoutManager ([commit](https://github.com/facebook/react-native/commit/ee597bfe2bf7f11ac60dab87f47dbcb6e840749c)).
```

Reply:
We are using capital letters for proper modules (Animation, Pressable, Image) and lower case for the rest (infra, text engine, runtime).